go.mod

```diff
@@ -25,4 +25,6 @@ require (
 	gopkg.in/macaroon.v2 v2.1.0
 )
 
+replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.11.1-beta
```

Comment:
is this because of the implicit loop dependency?

Reply:
Yeah, both `loop` and `faraday` reference the `-beta.rc3` version. So without the replace directive, go mod changes it to that as well. Even though if semver was implemented correctly, the `-beta` should have precedence over `-beta.rc3`...